client/layout/masterbar/checkout.tsx

```diff
@@ -111,11 +113,23 @@ const CheckoutMasterbar: FunctionComponent< Props > = ( {
 			{ isHelpCenterEnabled && isHelpCenterVisible && (
 				<HelpCenter
 					content={
-						<InlineHelpCenterContent
-							selectedArticle={ selectedArticle }
-							setSelectedArticle={ setSelectedArticle }
-							setHelpCenterFooter={ setFooterContent }
-						/>
+						contactForm ? (
+							<ContactForm
+								mode={ contactForm }
+								onBackClick={ () => {
+									setOpenInContactPage( true );
+									setContactForm( null );
+								} }
+							/>
+						) : (
+							<InlineHelpCenterContent
+								selectedArticle={ selectedArticle }
+								setSelectedArticle={ setSelectedArticle }
+								setHelpCenterFooter={ setFooterContent }
+								setContactFormOpen={ setContactForm }
+								openInContactPage={ openInContactPage }
+							/>
+						)
```

Comment:
Eventually we'll have to make a react router (maybe hash router) at one point for sure.

Reply:
Yes, I was thinking about that indeed. Not right now, but this may become way too complicated 👍